aws/src/main/java/org/apache/iceberg/aws/dynamodb/DynamoDbCatalog.java

```diff
@@ -329,7 +329,7 @@ public boolean removeProperties(Namespace namespace, Set<String> properties)
   @Override
   public List<TableIdentifier> listTables(Namespace namespace) {
     List<TableIdentifier> identifiers = Lists.newArrayList();
-    Map<String, AttributeValue> lastEvaluatedKey;
+    Map<String, AttributeValue> lastEvaluatedKey = null;
```

Comment:
I don't think this change is needed? if not assigned it's default to null

Reply:
For local variables, if used without initialisation, compilation fails (using Jdk8). `error: variable lastEvaluatedKey might not have been initialized`

Reply:
I see, then it looks good to me